README.md

```diff
@@ -24,6 +24,24 @@ When iterating on your tests during development, simply use zuul `--local` mode
 
 See the [quickstart](https://github.com/defunctzombie/zuul/wiki/quickstart) page on the wiki for more details.
 
+### Automated browser tests
```

Comment:
Maybe "automated local"? I wanted to highlight that this isn't designed for debugging, and to distinguish it from `--local`.